src/database/database.js

```diff
@@ -1,20 +1,25 @@
 const Sequelize = require('sequelize')
 
+let useSSL = true
+
+// set use SSL to false if NODE_ENV is development or test
+if (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test') {
```

Comment:
does `if (process.env.NODE_ENV != 'production')` work?

Reply:
Good catch, I will change it.